samples/net/sockets/raw/README.rst

```diff
@@ -0,0 +1,27 @@
+.. _raw-socket-sample:
+
+Raw socket sample
+#################
+
+Overview
+********
+
+This sample is a simple RAW socket application showing usage of
+raw socket over ethernet. Sample prints every packet which
+etherent receives and sends every packet over the wire. Zephyr
+network subsystem does not touch any of the headers(L2, L3 and
+so on).
+
+Building and Running
+********************
+
+When the application runs, it opens a raw socket and prints
+the length of packet it receives. And sends dummy packet over
+the wire at a rate every 5 seconds.
+
+See the `net-tools`_ project for more details.
+
+This sample can be built and executed on QEMU as described
+in :ref:`networking_with_qemu`.
```

Comment:
I guess it's not enough to tell just that. I guess it should say that outgoing packets can be observed with Wireshark (or if there's "easier" way, it should mention that).